src/commands/wavy.js

```diff
@@ -1,9 +1,27 @@
+import axios from 'axios'
+import Discord from 'discord.js'
 import DiscordCommand from './command.js';
 
 export default class WavyCommand extends DiscordCommand {
-    async execute() {
-        const wavy = this.getEmojiByName('wavy');
-        const wavyLeft = this.getEmojiByName('wavyLeft');
-        this.message.channel.send(`${wavy} ${wavyLeft}`);
+  async execute() {
+    if (!this.member) {
+      this.send("You must be in a Roo Bot enabled Discord server to use this command");
+      return;
     }
+
+    const wavy = this.getEmojiByName('wavy');
+    this.message.react(wavy);
+
+    axios.get(
+      `http://api.giphy.com/v1/gifs/translate?s=wavy&api_key=${process.env.GIPHY_API_KEY}`
+    ).then(response => {
+      const wavyEmbed = new Discord.MessageEmbed()
+        .setColor(this.member.displayHexColor || '#FFF')
+        .setAuthor(`${this.member.displayName} be hella wavy`, this.member.user.avatarURL())
+        .setThumbnail(wavy.url)
+        .setImage(`https://media.giphy.com/media/${response.data.data.id}/giphy.gif`);
```

Comment:
i'd probably extract this into a method that constructs the embed and returns it, just to break up the logic a bit.